.github/actions/slack/action.yaml

```diff
@@ -9,23 +9,12 @@ inputs:
   env_name:
     description: Name of env involved - one of dev, test, stage or prod
     required: true
-    type: choice
-    options:
-      - dev
-      - test
-      - stage
-      - prod
```

Comment:
Heya - were these changes in action.yaml deliberately backed out, or does this branch need a rebase on `main`?